### PR TITLE
Computes Murmur's height dynamically

### PR DESCRIPTION
**Pull request by acadet**
[Relative to this issue](https://github.com/hyperoslo/Whisper/issues/51)

dynamic murmur

### CODE REVIEW

**Comment by zenangst** [comment]
This looks neat! What do you guys think? @RamonGilabert @vadymmarkov

**Comment by aimak** [comment]
This is super useful :)

**Comment by acadet** [comment]
Thanks guys :)
So... when can we merge it?

**Comment by LucidityDesign** [comment]
+1

**Comment by RamonGilabert** [comment]
Hey @acadet, sorry about the super long delay. Been some crazy days at the office.

I see two problems in the PR, one is the indentation, easy fix, we are using a 2 indentation for the projects, the PR I see is 4, just a small thing! :)

The other thing is that you "kill" the constant height, that means that, if the sizeToFit is smaller than the status bar, Whistle will be smaller, would it? You could put like a check if it's smaller to use the dimensions.

What do you think?

File: Source/WhistleFactory.swift [hunk 1]
```diff
@@ -74,10 +72,24 @@ public class WhistleFactory: UIViewController {
...
 
   public func setupFrames() {
-    titleLabel.sizeToFit()
+    let labelWidth = UIScreen.mainScreen().bounds.width
+    
+    if let text = titleLabel.text {
+        let neededDimensions =
+            NSString(string: text).boundingRectWithSize(
+                CGSize(width: labelWidth, height: CGFloat.infinity),
+                options: NSStringDrawingOptions.UsesLineFragmentOrigin,
```
Comment (RamonGilabert) [inline Source/WhistleFactory.swift]: Also NSStringDrawingOptions could be dropped.

**Comment by acadet** [comment]
Hey @RamonGilabert,

Sorry for the indentation, Swiftmat was messing with my code.
Let me know what you think of this fix.

**Comment by RamonGilabert** [comment]
Should be good now! :+1:

Nice work @acadet, we'll make a release soon to include this into a wiki.

Thank's for the pull request and again sorry about the late responses, been a crazy week. Keep them coming! :)

Ramon

**Comment by LucidityDesign** [comment]
Can we do the same for shouts?

**Pull request merged**